ninja/errors.py

```diff
@@ -66,10 +73,18 @@ def set_default_exc_handlers(api: "NinjaAPI") -> None:
         ValidationError,
         partial(_default_validation_error, api=api),
     )
+    api.add_exception_handler(
+        MethodNotAllowedError,
+        partial(_default_method_not_allowed, api=api),
+    )
     api.add_exception_handler(
         AuthenticationError,
         partial(_default_authentication_error, api=api),
     )
+    api.add_exception_handler(
+        MethodNotAllowedError,
+        partial(_default_method_not_allowed, api=api),
+    )
```

Comment:
After your last push the `add_exception_handler` for a `MethodNotAllowedError` exception got duplicated.